app/controllers/policy_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+class DiscoursePolicy::PolicyController < ::ApplicationController
+  requires_plugin DiscoursePolicy::PLUGIN_NAME
+
+  before_action :ensure_logged_in, :set_post
+
+  def accept
+    PolicyUser.add!(current_user, @post.post_policy)
+    @post.publish_change_to_clients!(:policy_change)
+
+    render json: success_json
+  end
+
+  def unaccept
+    PolicyUser.remove!(current_user, @post.post_policy)
+    @post.publish_change_to_clients!(:policy_change)
+
+    render json: success_json
+  end
+
+  def accepted
+    users = @post
+      .post_policy
+      .accepted_by
+      .offset(params[:offset])
```

Comment:
```suggestion
      .offset(params[:offset].to_i)
```
Paranoid but it never hurts to avoid escaping!

Reply:
Rails automatically escapes `offset` parameter.